frontend_tests/node_tests/drafts.js

```diff
@@ -15,6 +15,7 @@ set_global("setTimeout", (f, delay) => {
     f();
 });
 const compose_state = mock_esm("../../static/js/compose_state");
+const people = mock_esm("../../static/js/people");
```

Comment:
Let's not mock the `people` module. You can just use `people.add_active_user` to setup the users you're going to use, like we do in most tests.

Reply:
Oh, I already noted this here: https://github.com/zulip/zulip/pull/22127/files#r944017296

Reply:
I'm pretty sure I'd have to change the whole test setup to be the newer kind to be able to use that, am I wrong about that?

Reply:
Not for `people`. The module didn't use `people.js` at all previously -- so you can just import it and add the handful of users you're going to use in the test, like most test modules do at the top.

(We also discussed removing the mock on `compose_state`, which would require refactoring of existing test code and is fine to defer)

Reply:
Oh I see! For some reason I'd ended up with the belief that this file wouldn't support `zrequire`, but I see how that's not true and there are several files that combine them. Alright, fixed :)

Reply:
Ahh, that's probably a generally useful fact to know, as if one doesn't, one is doomed to add unnecessary mocks :)